test-scenarios.md

```diff
@@ -0,0 +1,139 @@
+MLS Interop Test Scenarios
+==========================
+
+Participants are labeled A, B, C, ..., with the idea that they can be
+instantiated by any of the available implementations.  Tests are listed by
+scenario, which mainly corresponds to the message flow.  Other parameters
+include:
+
+* Ciphersuite
+* Credential type
+
+## Verifying Common State (subroutine)
+
+Since the major goal of MLS is to synchronize cryptographic state across a
+group's members, the key factor for verifying interop is verifying whether
+clients do indeed have the same cryptographic state.  The following things
+should be checked:
+
+* Each member's state produces the same authentication secret
+* An export with the same label and context produces the same exported value at
+  each member
```

Comment:
I think the export secret and the authentication secret are somewhat redundant. It would be enough to test them once and then only use one of them here.

Reply:
Yeah, this seems sensible.  We can validate the correctness of the export logic in test vectors (which I have added to the test harness interface).